lib/src/transformations/approximate_time.dart

```diff
@@ -144,7 +143,7 @@ class ApproximateTimeTransformation extends Transformation<Duration, String> {
       final fraction = primaryValue - primaryValue.truncate();
       var secondaryQuantifier = _SecondaryQuantifier.none;
 
-      if (fraction != Decimal.zero) {
+      if (fraction != Decimals.zero) {
```

Comment:
Got sick of seeing `Decimal.$constant` in some places and `Decimals.$constant` in other, so normalized all into `Decimals`.